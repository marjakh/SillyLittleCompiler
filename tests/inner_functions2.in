let var_outer = 56;

function f1() {
  let var_middle = 67;
  write(var_outer);
  write(var_middle);
  function f2() {
    let var_inner = 78;
    write(var_outer);
    write(var_middle);
    write(var_inner);
  }
  f2();
}

f1();