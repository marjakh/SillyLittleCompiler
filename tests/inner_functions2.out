56
67
56
67
78
